cmd/tasks/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	brokertags "github.com/cloud-gov/go-broker-tags"
+	config "github.com/cloud-gov/s3-broker/cmd/tasks/config"
+	tasksS3 "github.com/cloud-gov/s3-broker/cmd/tasks/s3"
+	cf "github.com/cloudfoundry/go-cfclient/v3/client"
+	cfconfig "github.com/cloudfoundry/go-cfclient/v3/config"
+
+	"golang.org/x/exp/slices"
+)
+
+type serviceNames []string
+
+// String is an implementation of the flag.Value interface
+func (s *serviceNames) String() string {
+	return fmt.Sprintf("%v", *s)
+}
+
+// Set is an implementation of the flag.Value interface
+func (s *serviceNames) Set(value string) error {
+	*s = append(*s, value)
+	return nil
+}
+
+var servicesToTag serviceNames
+
+var (
+	configFilePath string
+)
+
+func init() {
+	flag.StringVar(&configFilePath, "config", "", "Location of the config file")
+}
+
+func run() error {
+	actionPtr := flag.String("action", "", "Action to take. Accepted options: 'reconcile-tags'")
+	flag.Var(&servicesToTag, "service", "Specify AWS service whose instances should have tags updated. Accepted options: 's3'")
```

Comment:
We don't need this option in this broker. Since the only supported service is S3